packages/react-dom-bindings/src/client/ReactDOMComponent.js

```diff
@@ -1790,7 +1788,7 @@ function diffHydratedCustomComponent(
           }
           continue;
```

Comment:
is this `continue` meant to be unconditional?

Reply:
It's supposed to only be a continue if the `enableCustomElementPropertySupport` flag is on, otherwise fallthrough as if they weren't special cased.

Reply:
ohhhhhhhh the 'className' case is empty if enableCustomElementPropertySupport is false. got it

Reply:
I end up using this technique of putting fall throughs at the end so that the cases themselves get DCE.